vst2_windows.go

```diff
@@ -30,34 +29,35 @@ type handle struct {
 func init() {
 	envVstPath := os.Getenv("VST_PATH")
 	if len(envVstPath) > 0 {
-		scanPaths = append(paths, envVstPath)
+		scanPaths = append(scanPaths, envVstPath)
 	}
 }
 
 // open loads the plugin entry point into memory. It's DLL in windows.
 func open(path string) (effectMain, handle, error) {
 	//Load plugin by path
-	dll, err := syscall.LoadDLL(l.Path)
+	dll, err := syscall.LoadDLL(path)
 	if err != nil {
-		return nil, handle{}, fmt.Errorf("failed to load VST from '%s': %v\n", l.Path, err)
+		return nil, handle{}, fmt.Errorf("failed to load VST from '%s': %v\n", path, err)
 	}
-	l.library = unsafe.Pointer(dll)
-	l.Name = strings.TrimSuffix(filepath.Base(dll.Name), filepath.Ext(dll.Name))
+	//l.library = unsafe.Pointer(dll) ???
+	//l.Name = strings.TrimSuffix(filepath.Base(dll.Name), filepath.Ext(dll.Name))
 
 	//Get pointer to plugin's Main function
 	m, err := syscall.GetProcAddress(dll.Handle, main)
 	if err != nil {
-		l.Close()
-		return nil, handle{}, fmt.Errorf("failed to get entry point for plugin'%s': %v\n", l.Path, err)
+		//l.Close() ???
```

Comment:
I think that in this case the `dll.Release()` should be executed.

Reply:
added, but ignoring the err explicitly. Perhaps it will be better to check the err.